test/unit/spec/FidoUtil_spec.js

```diff
@@ -0,0 +1,11 @@
+define([
+  'helpers/util/Expect',
+  'util/FidoUtil'
+],
+function (Expect, FidoUtil) {
+  Expect.describe('FidoUtil', function () {
+    it('Gets U2F version', function () {
+      expect(FidoUtil.getU2fVersion()).toEqual('U2F_V2');
+    });
+  });
```

Comment:
I know it was just a refactor, but can we add a couple of simple tests for the other 2 functions in FidoUtil?